Tasks/DockerV2/dockerlifecycle.ts

```diff
@@ -0,0 +1,38 @@
+"use strict";
+
+import * as tl from "azure-pipelines-task-lib/task";
+import ContainerConnection from "docker-common-v2/containerconnection";
+import * as dockerCommandUtils from "docker-common-v2/dockercommandutils";
+import * as utils from "./utils";
+
+export function run(connection: ContainerConnection, outputUpdate: (data: string) => any): any {
+    let output = "";
+    let command = tl.getInput("command", true);
+    let containerName = tl.getInput("container", true);
+    let commandArguments = dockerCommandUtils.getCommandArguments(tl.getInput("arguments", false));
+
+    let containerMap = tl.getVariable("agent.containermapping");
+    if (!containerMap) {
+        throw new Error(tl.loc('ContainerNotFound', containerName));
+    }
+
+    let map = JSON.parse(containerMap);
+    let containerId = map[containerName].id;
+    if (!containerId || containerId == "") {
```

Comment:
Remove `|| containerId == ""`.
`!containerId` will take care of empty string too.